### ISSUE TITLE
Suggest moving the repository to electron-userland

### ISSUE THREAD

**Issue by inukshuk** [jprichardson/electron-mocha#185]
@jprichardson would it be OK with you to request if the repository could be transferred to [electron-userland](https://github.com/electron-userland)? The main reason I ask is because we'll need to update the Travis CI integration or convert to GitHub Actions for which it would be helpful for me to have admin access.

**Comment by jprichardson** [comment]
Yes, absolutely. I can also make you admin here if that’s possible. Whichever you prefer.

**Comment by inukshuk** [comment]
Great! I'll open a request over at electron-userland to see what they think.
